Src/CSharpier/DocPrinter/DocPrinter.cs

```diff
@@ -56,6 +53,21 @@ public string Print()
             return result;
         }
 
+        private void EnsureOutputEndsWithSingleNewLine()
+        {
+            var trimmed = 0;
+            for (; trimmed < Output.Length; trimmed++)
+            {
+                if (Output[^(trimmed + 1)] != '\r' && Output[^(trimmed + 1)] != '\n')
```

Comment:
Does this work for Unix line endings that don't have a carriage return?

Reply:
It does, I have some unit tests for it in DocPrinterTests. I did have a copy paste error and wasn't actually testing just "\n" though.